.gitmodules

```diff
@@ -35,3 +35,7 @@
 [submodule "extern/libusb/src"]
 	path = extern/libusb/src
 	url = https://github.com/libusb/libusb
+[submodule "extern/libuvc/src"]
+	path = extern/libuvc/src
+	url = https://github.com/wes-b/libuvc.git
```

Comment:
>wes-b [](start = 26, length = 5)

We might want to have the libuvc fork owned by the Microsoft corporate account to increase our bus factor

Reply:
I agree with this long term. In the short term I have pushed a PR back to the owner to see if I get any traction with them taking a fix. There is not too much action over at https://github.com/ktossell/libuvc, so if that doesn't work out we will work to get this forked under Microsoft. For now this change is unblocked while we explore our libuvc options.

Reply:
Ah, okay, I didn't realize the fork was a temporary thing pending a PR being accepted.  In that case, this is fine.

---
In reply to: [284937090](https://github.com/microsoft/Azure-Kinect-Sensor-SDK/pull/350#discussion_r284937090) [](ancestors = 284937090)